docs/15_EgenskaperInformasjonsmodell.adoc

```diff
@@ -188,18 +203,30 @@ Klassen _Informasjonsmodell_ er obligatorisk
 
 [cols="30s,70d"]
 |===
+|Engelsk navn | replaces
 |URI|dct:replaces
 |Range|modelldcatno:InformationModel
 |Beskrivelse|Referanse til eldre utgått modell denne modellen er ment å erstatte.
 |Multiplisitet|0..n
 |Status|Valgfri
 |===
 
+===== Informasjonsmodell:finnes i format (dct:hasFormat) [[informasjonsmodell-finnesIFormat]]
+
+|===
+|Engelsk navn|has format
+|URI|dct:hasFormat
+|Range|foaf:Document
+|Beskrivelse|Brukes til å referere til et dokument som representerer modellen i et annet format.
+|Multiplisitet|0..n
+|Status|Valgfri
+|===
 
 ===== Informasjonsmodell: gyldighetsperiode (dct:temporal) [[informasjonsmodell-gyldighetsperiode]]
 
 [cols="30s,70d"]
 |===
+|Engelsk navn | temopral
 |URI|dct:temporal
 |Range|dct:PeriodeOfTime
```

Comment:
Fant at PeriodOfTime også hadde en e for mye her også.

Reply:
> Fant at PeriodOfTime også hadde en e for mye her også.

Rettet, ny commit.